Extension/src/Debugger/extension.ts

```diff
@@ -24,6 +24,26 @@ export function buildAndDebugActiveFileStr(): string {
     return ` - ${localize("build.and.debug.active.file", 'Build and debug active file')}`;
 }
 
+/*
+   Separate initialize method for Alpine until we can use the regular 'initialize' method.
+
+   This enables attach and cppdbg for alpine without snippets or 'Build and Debug Active file'
+   since we do not have a language service.
```

Comment:
Do we want one?  What would it take for us to make it work?  It would be better if we could just support the platform across the board.

Reply:
Lets discuss this in our next meeting.

Reply:
Well the current one is glibc-based.